src/main/java/Gandalf.java

```diff
@@ -0,0 +1,183 @@
+import java.util.Scanner;
+
+public class Gandalf {
+    public static final String LINE = "____________________________________________________________";
+
+    public static void makeList() {
+        makeListWelcomeMessage();
+
+        Task[] listTasks = new Task[100];
+        Scanner in = new Scanner(System.in);
+        String text = in.nextLine();
+
+        int insertIndex = 0;
+
+        while (true) {
+            if (text.equals("bye")) {
+                System.out.println(LINE);
+                return;
+            } else if (text.equals("list")) {
```

Comment:
Using enumerations for the task types can help reduce the number of if-else statements.